sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/LocalKeyCryptographyClient.java

```diff
@@ -27,21 +27,22 @@
         this.serviceClient = serviceClient;
     }
 
-    abstract Mono<EncryptResult> encryptAsync(EncryptionAlgorithm algorithm, byte[] plaintext, Context context, JsonWebKey jsonWebKey);
+    abstract Mono<EncryptResult> encryptAsync(EncryptionAlgorithm algorithm, byte[] plaintext,
```

Comment:
Comments about using the `EncryptOptions` and similar should apply to the `LocalCryptographyClient`. In fact, I'm just noticing this says "LocalKeyCryptographyClient". It should be "LocalCryptographyClient" (that's what we shipped for betas for .NET and JS).

Reply:
Actually, I see this isn't exposed. But I'll leave the comment here in case it still applies.